src/content/docs/apis/rest-api-v2/get-started/introduction-new-relic-rest-api-v2.mdx

```diff
@@ -84,6 +84,6 @@ The <var>&lt;PAYLOAD></var> contains the query details, which define:
 See the following docs for example REST API use cases:
 
 * [APM examples](/docs/apis/rest-api-v2/application-examples-v2) (how to retrieve metric timeslice data from APM)
-* [Browser examples](/docs/apis/rest-api-v2/browser-examples-v2) (how to retrieve metric timeslice data from browser monitoring)
+* [Browser examples](/docs/apis/rest-api-v2/browser-examples-v2) (how to retrieve metric timeslice data from browser monitoring). To note, the REST API will only return the Lite agent script even though newly created browser apps (either via APM auto-inject or manual, copy/paste installs) are set to serve up Pro+SPA agent as the default.
```

Comment:
```suggestion
* [Browser examples](/docs/apis/rest-api-v2/browser-examples-v2) (how to retrieve metric timeslice data from browser monitoring). The REST API only returns the Lite browser agent script. For more information, [see this table comparing Lite, Pro, and Pro+SPA types](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent).
```

How about something like this @CherylFrankenfield? Instead of trying to explain it in this bulleted list, we can link to the table comparing these three types.

Reply:
@paperclypse Looks great! Thanks for the fast turnaround!